pkg/phlare/phlare.go

```diff
@@ -492,7 +492,7 @@ func initLogger(logFormat string, logLevel dslog.Level) log.Logger {
 }
 
 func (f *Phlare) initAPI() (services.Service, error) {
-	a, err := api.New(f.Cfg.API, f.Server, f.grpcGatewayMux, util.Logger)
```

Comment:
this util.Logger is NoOpLogger

Reply:
That's annoying, thanks for spotting

Reply:
we should rename util.Logger to NoOpLoggerPleasNoUseIt or  remove it. there's still 27 usages of it
<img width="1099" alt="Screenshot 2023-09-21 at 7 27 52 PM" src="https://github.com/grafana/pyroscope/assets/331773/318119d1-31b1-49c3-b8f7-5795a34a99ad">

Reply:
Yeah Mimir uses that global, can you create an issues to try to get rid of it.

Reply:
Yeah already created 
https://github.com/grafana/pyroscope/issues/2435

> eah Mimir uses that globa
o_O mimir? it is pyroscope/pkg/util/logger.go

Reply:
😆 we obviously copy pasted it all from mimir